chains/milkomeda-c1-testnet.json

```diff
@@ -19,6 +19,6 @@
       "rpcUrl": "https://rpc-devnet-cardano-evm.c1.milkomeda.com"
     }
   ],
-  "symbol": "testmilkADA",
+  "symbol": "milkADA",
```

Comment:
We can also use `mADA` as per https://docs.milkomeda.com/cardano/for-end-users/obtaining-milkada/ Alternatively, we can just call it ADA because we call all L2 ETH ETH and not anything else (like [oETH](https://optifaucet.com/)).